engine/apps/alerts/escalation_snapshot/snapshot_classes/escalation_snapshot.py

```diff
@@ -43,7 +44,7 @@ def __init__(
         escalation_policies_snapshots: typing.List["EscalationPolicySnapshot"],
         slack_channel_id: str,
         pause_escalation: bool,
-        next_step_eta: typing.Optional[str],
+        next_step_eta: typing.Optional[datetime.datetime],
```

Comment:
the typing here was wrong. This can't possibly be a `str` since we are doing the following down on line 106:
```python
return self.next_step_eta > (timezone.now() - datetime.timedelta(minutes=5))
```